uaclient/tests/test_config.py

```diff
@@ -444,6 +452,9 @@ def test_root_unattached(
         assert expected == cfg.status(show_beta=show_beta)
 
     @pytest.mark.parametrize("show_beta", (True, False))
+    @pytest.mark.parametrize(
+        "features_override", ((None), ({"features": {"allow_beta": True}}))
```

Comment:
Per the above suggestion
```suggestion
        "features_override", ((None), ({"allow_beta": True}))
```